docs/changelogs/v0.15.md

```diff
@@ -0,0 +1,334 @@
+# Kubo changelog v0.15
+
+## v0.15.0
+
+### Overview
+
+Below is an outline of all that is in this release, so you get a sense of all that's included.
+
+- [Kubo changelog v0.15](#kubo-changelog-v015)
+  - [v0.15.0](#v0150)
+    - [Overview](#overview)
+    - [🔦 Highlights](#-highlights)
+      - [#️⃣ Blake support](#️⃣-blake-support)
+      - [💉 Fx Options plugin](#-fx-options-plugin)
+    - [Changelog](#changelog)
+    - [Contributors](#contributors)
+
+
+### 🔦 Highlights
+
+This is a release mainly with bugfixing and library updates. 
+We are improving release speed and cadence trying to have a new release every 5 weeks.
+
+#### #️⃣ Blake support
+
+You can now use `blake3` as a valid hash function: `ipfs block put --mhtype=blake3`.
+It uses a 32 bytes default size.
+To use a different hash length, set `mhlen`: `ipfs block put --mhtype=blake3 --mhlen=64`
```

Comment:
```suggestion
You can now use `blake3` as a valid hash function:
- `ipfs block put --mhtype=blake3`
- `ipfs add --hash=blake3`
It uses a 32 bytes default size.
And verify up to 128 bytes.
Because `blake3` is variable output hash function, you can use a different digest length, set `mhlen`: `ipfs block put --mhtype=blake3 --mhlen=64`, `ipfs add` doesn't have this option yet.
```